CAM/InvalidateVehicleIdleCam.md

```diff
@@ -0,0 +1,23 @@
+---
+ns: CAM
+aliases: ["0x9E4CFFF989258472"]
+---
+## _INVALIDATE_VEHICLE_IDLE_CAM
+
+```c
+// 0x9E4CFFF989258472 0x65DDE8AF
+void INVALIDATE_VEHICLE_IDLE_CAM();
```

Comment:
This line needs prefix too 😅
See other natives for example: https://github.com/citizenfx/natives/blob/master/LOADINGSCREEN/LoadingscreenSetLoadFreemode.md

Reply:
Oh no I missed it completly! Thought there only was one